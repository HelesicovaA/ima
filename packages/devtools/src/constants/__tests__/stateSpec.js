import State from '../state';

describe('state constant', () => {
  it('should match snapshot', () => {
    expect(State).toMatchSnapshot();
  });
});
