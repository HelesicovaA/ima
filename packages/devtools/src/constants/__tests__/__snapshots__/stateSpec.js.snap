// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`State constant should match snapshot 1`] = `
Object {
  "ALIVE": "alive",
  "DEAD": "dead",
  "DETECTING": "detecting",
  "RELOAD": "reload",
}
`;
